src/Sylius/Bundle/CoreBundle/Command/ClearPriceHistoryCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Command;
```

Comment:
Move this service (and others) to PriceHistory namespace in CoreBundle
```suggestion
namespace Sylius\Bundle\CoreBundle\PriceHistory\Command;
```